imports/test-utils/helpers/getFakeMongoCursor.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @name getFakeMongoCursor
+ * @param {String} collectionName - name of the collection
+ * @param {Any} results - results to be returned as part of the cursor
+ * @return {Object} fake cursor
+ */
+export default function getFakeMongoCursor(collectionName, results, options) {
+  const cursor = {
+    cmd: {
+      query: (options && options.query) || {}
+    },
+    ns: `meteor.${collectionName}`,
+    options: {
+      db: {
+        databaseName: "meteor"
+      }
+    }
+  };
+
+  const isTesting = typeof jest !== "undefined";
+
+  if (isTesting) {
+    cursor.clone = jest.fn().mockName("cursor.clone").mockReturnValue({
+      count: jest.fn().mockName("cursor.clone.count").mockReturnValueOnce(results.length)
+    });
```

Comment:
Change this to

```js
cursor.clone = jest.fn().mockName("cursor.clone").mockImplementation(() => cursor);
```

And then add:

```js
cursor.count = jest.fn().mockName("cursor.count").mockReturnValue(results.length)
```

And then you don't have to do `mockCursor.clone = () => ({ count: mockCount });` in `applyPaginationToMongoCursor.test.js`

(Make the same change in the non-jest code. Have clone return `cursor` and add `cursor.count` func.)